packages/camera/camera_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.1
+
+* Update to latest camera platform interface, and fail if user attempts to use streaming with recording (since streaming is currently unsupported on web).
```

Comment:
Nit: "Updates" and "fails" (per the CHANGELOG style guide linked from the PR checklist).

Reply:
Done